### ISSUE TITLE
Please add these options

### ISSUE THREAD

**Issue by dillfrescott** [starrytong/SCNet#22]
I really sloppily added (or attempted to add) the prodigy optimizer and also auraloss [here](https://github.com/starrytong/SCNet/compare/main...dillfrescott:SCNet:main).

Would you mind taking a look at the changes and adding the option to use prodigy and auraloss a better way in the official repo? I apologize as my coding knowledge is limited and I am not much help.

I would love to have these two things officially added as config options!

**Comment by starrytong** [comment]
Do prodigy and auraloss help with training? In your experiments, were they able to effectively improve performance?

**Comment by dillfrescott** [comment]
Prodigy basically handles the learning rate for you, and I've heard that it greatly improves performance. Auraloss doesnt necessarily improve training performance but it is more akin to human hearing and can give much better results than MSE or L1 loss functions. It can sort of repair the instrumental after the vocal removal to an extent.

**Comment by dillfrescott** [comment]
never mind they are already added in zfturbos training repo which has scnet so Ill close this.
